frontend/src/components/header/Header.tsx

```diff
@@ -37,21 +35,9 @@ const Header: React.FC<HeaderProps> = ({isLogged}: HeaderProps) => {
 
     const findRouteByPath = (path: string) => {
         const normalizedPath = path.toLowerCase().replace(/\s/g, '');
-        return mainRoutes.find((route: IRoute) => route.path.trim().slice(1).toLowerCase() === normalizedPath);
+        return mainClientRoutes.find((route: IRoute) => route.path.trim().slice(1).toLowerCase() === normalizedPath);
```

Comment:
I would make it more generic by selecting routes based on user type. Who knows, maybe merchant will have more tabs in header later